package.json

```diff
@@ -9,20 +9,30 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@date-io/date-fns": "^2.11.0",
+    "@date-io/moment": "1.x",
     "@material-ui/core": "^4.12.3",
     "@material-ui/icons": "^4.11.2",
+    "@material-ui/lab": "^4.0.0-alpha.60",
+    "@material-ui/pickers": "^3.3.10",
     "@types/color": "^3.0.2",
     "@types/react-helmet": "^6.1.2",
     "classnames": "^2.3.1",
     "color": "^4.0.0",
     "emailjs-com": "^3.2.0",
     "fetch-jsonp": "^1.2.1",
+    "moment": "^2.29.1",
     "next": "11.1.1",
+    "next-redux-wrapper": "^7.0.5",
     "react": "17.0.2",
     "react-dom": "17.0.2",
     "react-helmet": "^6.1.0",
     "react-hook-form": "^7.14.0",
-    "react-scrollmagic": "^2.3.0"
+    "react-redux": "^7.2.5",
+    "react-scrollmagic": "^2.3.0",
+    "redux": "^4.1.1",
```

Comment:
Pls feel free to not answer this question if you don't have time since I know you are busy!

Out of curiosity, why'd you choose redux for the website? Not hating on the choice(I don't care haha) but was curious on the design choice.